.gitignore

```diff
@@ -252,3 +252,4 @@ ModelManifest.xml
 /src/Stratis.Bitcoin.Tests/Stratis.Bitcoin.Tests/TestData
 /src/Stratis.StratisD/Logs
 /src/Stratis.BitcoinD/Logs
+src/Stratis.Bitcoin.Tests/Consensus/ChainedHeaderTreeTest.cs.orig
```

Comment:
Please remove and check this out:

https://stackoverflow.com/questions/1251681/git-mergetool-generates-unwanted-orig-files

Reply:
oopps sorry...missed that one